### ISSUE TITLE
Bug with timezones for entries around midnight

### ISSUE THREAD

**Issue by enrico3** [dartungar/obsidian-mood-tracker#53]
Hi,

I added an entry at 2024-06-03 1:37 am. My timezone offset is +2:
![grafik](https://github.com/dartungar/obsidian-mood-tracker/assets/31779179/01409218-ca80-409d-835f-407d158216c1)

The graph in Obsidian shows this entry for 2024-06-02 and does not show the details below:
![grafik](https://github.com/dartungar/obsidian-mood-tracker/assets/31779179/de51ef3b-9464-48e1-8960-a46dc386340d)

If I change the timestamp to say 3:37am then it works:
![grafik](https://github.com/dartungar/obsidian-mood-tracker/assets/31779179/3e780e57-ef0b-4702-8a66-b5fdd4d478bd)
![grafik](https://github.com/dartungar/obsidian-mood-tracker/assets/31779179/2bfa603c-0166-4e41-ba44-71be2ba204dd)

Cheers!

**Comment by dartungar** [comment]
@enrico3 this has to do with ChartJS date handling. I haven't found a good solution yet; for now, you might want to work around that by adjusting time of logging (hopefully this won't be a big issue since the problem only occurs after the midnight...).

**Comment by enrico3** [comment]
@dartungar Yeah for me it's not a problem. I just wanted to let you know. Thank you for the info!
